src/debug.ts

```diff
@@ -106,16 +104,40 @@ async function createTestConfigurations(ctx: FolderContext): Promise<vscode.Debu
             cwd: `\${workspaceFolder:${ctx.folder.name}}`,
             preLaunchTask: `swift: Build All`
         }];
+    } else if (process.platform === 'win32') {
+        // On Windows, add XCTest.dll to the PATH,
+        // and then run the .xctest bundle from the .build directory.
+        let xctestPath: string;
+        try {
+            xctestPath = await getXCTestDLLPath();
+        } catch (error) {
+            vscode.window.showErrorMessage(
```

Comment:
I could store it in the workspace context, since it's a global setting?
But the `createTestConfigurations` function only has access to the folder context, not the workspace context. I'd really like to have a back-reference to the workspace context in each folder context.  I don't even think we need to worry about reference cycles here, due to how the garbage collector works, but I'd need to double-check that.

Reply:
Looks like cyclical references aren't a concern anymore with modern JS implementations so maybe it isn't an issue in the end. Add a `WorkspaceContext` reference into the `FolderContext`. It will probably make our life easier.

Reply:
Done